rules/linux/execution_shell_evasion_linux_binary.toml

```diff
@@ -97,64 +97,76 @@ references = [
-        (process.parent.name in ("byebug","git","ftp","strace","nawk", "mawk", "awk", "gawk", "tar", "zip")) or
-
-        /* shells specified in parent args */
-        /* nice rule is broken in 8.2 */
-        (process.parent.args in ("/bin/sh", "/bin/bash", "/bin/dash", "/bin/ash", "sh", "bash", "dash", "ash") and
-            (
-             (process.parent.name == "nice") or
-             (process.parent.name == "cpulimit" and process.parent.args == "-f") or
-             (process.parent.name == "find" and process.parent.args == "-exec" and process.parent.args == ";") or
-             (process.parent.name == "flock" and process.parent.args == "-u" and process.parent.args == "/")
-            )
-        ) or
-
-         /* shells specified in args */
-         (process.args in ("/bin/sh", "/bin/bash", "/bin/dash", "/bin/ash", "sh", "bash", "dash", "ash") and
-            (process.parent.name == "crash" and process.parent.args == "-h") or
-            (process.name == "sensible-pager" and process.parent.name in ("apt", "apt-get") and process.parent.args == "changelog")
-            /* scope to include more sensible-pager invoked shells with different parent process to reduce noise and remove false positives */
-          )
-    ) or
-    (process.name == "busybox" and process.args_count == 2 and process.args in ("/bin/sh", "/bin/bash", "/bin/dash", "/bin/ash", "sh", "bash", "dash", "ash") )or
-    (process.name == "env" and process.args_count == 2 and process.args in ("/bin/sh", "/bin/bash", "/bin/dash", "/bin/ash", "sh", "bash", "dash", "ash")) or
-    (process.parent.name in ("vi", "vim") and process.parent.args == "-c" and process.parent.args in (":!/bin/bash", ":!/bin/sh", ":!bash", ":!sh")) or
-    (process.parent.name in ("c89","c99", "gcc") and process.parent.args in ("sh,-s", "bash,-s", "dash,-s", "ash,-s", "/bin/sh,-s", "/bin/bash,-s", "/bin/dash,-s", "/bin/ash,-s") and process.parent.args == "-wrapper") or
-    (process.parent.name == "expect" and process.parent.args == "-c" and process.parent.args in ("spawn /bin/sh;interact", "spawn /bin/bash;interact", "spawn /bin/dash;interact", "spawn sh;interact", "spawn bash;interact", "spawn dash;interact")) or
-    (process.parent.name == "mysql" and process.parent.args == "-e" and process.parent.args in ("\\!*sh", "\\!*bash", "\\!*dash", "\\!*/bin/sh", "\\!*/bin/bash", "\\!*/bin/dash")) or
-    (process.parent.name == "ssh" and process.parent.args == "-o" and process.parent.args in ("ProxyCommand=;sh 0<&2 1>&2", "ProxyCommand=;bash 0<&2 1>&2", "ProxyCommand=;dash 0<&2 1>&2", "ProxyCommand=;/bin/sh 0<&2 1>&2", "ProxyCommand=;/bin/bash 0<&2 1>&2", "ProxyCommand=;/bin/dash 0<&2 1>&2"))
-  )
+(
+  /* launching shell from capsh */
+  (process.name == "capsh" and process.args == "--") or
+  
+  /* launching shell from awk, nawk, mawk or gawk */
+  (process.name : "*sh" and process.parent.name : "*awk" and process.parent.args : "BEGIN {system(*)}") or
```

Comment:
@Aegrah we can still get this under one umbrella of process name
(process.name : "*sh") and
{
   // new refining from tuning 
}